porting/index.md

```diff
@@ -156,6 +219,12 @@ Here you need to choose your device `cm_[your device]-userdebug`, example if you
 
 ### Building the system.img and hybris-boot.img
 
+Halium will use the mkbootimg tool for creating the boot image. In most cases it is not on the local harddisk, so it can be built by issuing
```

Comment:
Lets leave it for the moment, it was a bit confusign to me so others may benefit. We can take it out later again.